test/runnativetests.cmd

```diff
@@ -179,7 +179,8 @@ goto :main
 :runtest
   pushd %_TestTempDir%
   call :do nativetests.exe
-  if ERRORLEVEL 1 set _HadFailures=1
+  if %ERRORLEVEL% NEQ 0 set _HadFailures=1
+  echo -- runnativetests.cmd ^>^> nativetests.exe exiting with exit code %_HadFailures%

```

Comment:
Is there a way to access test logs of `nativetests.exe`. Currently I don't see a log for these test.